orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccountInterventionService.java

```diff
@@ -139,13 +139,19 @@ private static AuditContext addStatusMetadata(
     private AccountIntervention handleException(Exception e) {
         cloudwatchMetricsService.incrementCounter(
                 configurationService.getAccountInterventionsErrorMetricName(),
-                Map.of("Environment", configurationService.getEnvironment()));
+                Map.of(
+                        "Environment",
+                        configurationService.getEnvironment(),
+                        "AbortOnError",
+                        String.valueOf(acountInterventionsAbortOnError)));
         if (accountInterventionsActionEnabled && acountInterventionsAbortOnError) {
-            throw new AccountInterventionException(
-                    "Problem communicating with Account Intervention Service", e);
+            String errorMessage =
+                    "Problem communicating with Account Intervention Service. Aborting user journey.";
+            LOG.error(errorMessage, e);
```

Comment:
This does mean that the other alarm in this lambda is also probably going to go off at the same time if we P1 - but that's not the end of the world, and I think we can look into tidying that up when we pick up further alarms work

Reply:
Yeah it will, better too many alarms that too few 😅 . Another thing to look at in the general alarms audit.